extensions/ql-vscode/src/interface.ts

```diff
@@ -384,14 +413,67 @@ export class InterfaceManager extends DisposableObject {
       resultsPath: this.convertPathToWebviewUri(
         results.query.resultsPaths.resultsPath
       ),
-      resultSets,
+      parsedResultSets: await getParsedResultSets(),
       sortedResultsMap,
       database: results.database,
       shouldKeepOldResultsWhileRendering,
       metadata: results.query.metadata
     });
   }
 
+  /**
+   * Show a page of raw results from the chosen table.
+   */
+  public async showPageOfResults(selectedTable: string, pageNumber: number): Promise<void> {
```

Comment:
I'd rather see `pageNumber` changed to `pageSize` and `offset`. For now, we can keep a single, default page size, but we should allow users to choose this in the future. It shouldn't be hard.